indico/modules/events/registration/templates/management/_reglist.html

```diff
@@ -80,7 +86,13 @@
                                                 -
                                             {% endif %}
                                         </td>
-                                    {% else %}
+                                    {% elif item.id == 'tags_present' %}
+                                        <td class="i-table">
+                                            {% for tag in registration.tags|sort(attribute='title', case_sensitive=false) %}
+                                                <span class="registration-tag" style="background-color: {{ tag.color }};">{{ tag.title }}</span>
```

Comment:
Now that we use the SUI colors, should we also use the SUI labels?

    <span class="ui label tiny {{ tag.color }}">{{ tag.title }}</span>

(not sure about the size; you; have to check which one fits best)